build.sbt

```diff
@@ -200,7 +200,8 @@ lazy val parboiled = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .jvmSettings(
     (Compile / packageBin / mappings) ++= (parboiledCoreJVM.project / Compile / packageBin / mappings).value,
     (Compile / packageSrc / mappings) ++= (parboiledCoreJVM.project / Compile / packageSrc / mappings).value,
-    (Compile / packageDoc / mappings) ++= (parboiledCoreJVM.project / Compile / packageDoc / mappings).value
+    (Compile / packageDoc / mappings) ++= (parboiledCoreJVM.project / Compile / packageDoc / mappings).value,
+    AutomaticModuleName.settings("org.parboiled2.parboiled")
```

Comment:
I think you should be able to simply say this here:

```scala
Compile / packageBin / packageOptions += "Automatic-Module-Name" ->""org.parboiled2.parboiled"
```
No need for the extra file with the object and everything.

Reply:
You're right I can simply it